src/basic/sysctl-util.c

```diff
@@ -44,6 +44,33 @@ char *sysctl_normalize(char *s) {
         return s;
 }
 
+char *sysctl_value_normalize(char *s) {
```

Comment:
what's this about?

Reply:
Added additional documentation.
So procfs tab separates multivalued properties - `/proc/sys/kernel/printk`, `/proc/sys/net/ipv4/tcp_rmem`. To facilitate this comparison, I added a value normalization.

Reply:
urks. i would be careful about that. we never know what we can do and what we can't. i'd be super careful, i.e. ignore trailing whitespace, not much else.

I mean, whether to use tabs or spaces is something the user relatively easily has control over, simply by writing the sysctl files the right way. The newline suffix is harder for the the user to control, since they cannot include it in the sysctl file, since we use newlines as separators while parsing lines after all.

Or to say that differently: this sounds like something worth documenting in the man page of sysctl.d, but not doing silently behind the user's back.